src/main/java/org/folio/repository/assigneduser/AssignedUserRepositoryImpl.java

```diff
@@ -122,6 +125,27 @@
     });
   }
 
+  @Override
+  public CompletableFuture<Void> delete(String credentialsId, String userId, String tenant) {
+    String query = format(DELETE_ASSIGNED_USER_QUERY, getAssignedUsersTableName(tenant));
+
+    LOG.info(DELETE_LOG_MESSAGE, query);
+    Promise<UpdateResult> promise = Promise.promise();
+    pgClient(tenant).execute(query, createParams(Arrays.asList(credentialsId, userId)), promise);
+
+    Future<UpdateResult> resultFuture = promise.future()
+      .recover(excTranslator.translateOrPassBy());
+    return mapResult(resultFuture, updateResult -> checkUserDeleted(userId, updateResult));
+  }
+
+  @Nullable
+  private Void checkUserDeleted(String userId, UpdateResult updateResult) {
```

Comment:
This method also can be used in `update` method

Reply:
added to `update` method